--- build_agent.py.orig
+++ build_agent.py
@@ -291,7 +291,7 @@
             f.write(r)
         
         # Run agent
-        subprocess.Popen([sys.executable, p, "--server", "{server_host}", "--port", "{server_port}"{"", "--persistence"' if with_persistence else ''}])
+        subprocess.Popen([sys.executable, p, "--server", server_host, "--port", str(server_port)] + (["--persistence"] if with_persistence else []))
     except Exception as e:
         pass
